modules/monitoring/files/check_reverse_dns.py

```diff
@@ -78,8 +78,14 @@ def check_records(hostname):
         for nameserver in nameserversans:
             nameserver = str(nameserver)
             nameservers.append(nameserver)
-            if nameserver.endswith('.ns.cloudflare.com.') or nameserver.endswith('.dreamhost.com.') or nameserver.endswith('.ns.porkbun.com.'):
-                uses_cf_at_root = True
+            flatten_manadatory_providers = [
+                'ns.cloudlfare.com',
+                'dreamhost.com',
+                'porkbun.com',
+            ]
+            for provider in flatten_manadatory_providers:
+                if nameserver.endswith(f'.{provider}'):
```

Comment:
```suggestion
                if nameserver.endswith(f'.{provider}.'):
```

Reply:
Also, endswith supports a tuple I think, so if list above is converted to a tuple, we do not need the loop I think?

https://docs.python.org/3/library/stdtypes.html#str.endswith

> suffix can also be a tuple of suffixes to look for.

Reply:
No reason why I can't make it a tuple

Reply:
This should be enough here also, to just add minor simplification I think, but that is up to you.
```lang=py
if nameserver.endswith(('.ns.cloudlfare.com.', '.dreamhost.com.', '.ns.porkbun.com.')):
    cname_check_impossible = True
```

Reply:
It's not simpler imho. Moving it to a tuple outside with each entry on a new line looks cleaner.

Reply:
Feel free to do however you feel best then.